apps/api/src/lessons/lessons.service.ts

```diff
@@ -163,6 +166,10 @@ export class LessonsService {
       userId,
       true,
     );
+
+    let correctAnswers = 0;
+    let wrongAnswers = 0;
```

Comment:
@wielopolski any ideas on how to get this data in a better way?

Reply:
maybe
```suggestion
    const correctAnswerCount = questionLessonItem.content.questionAnswers
  .filter(answer => Boolean(answer.isCorrect))
  .length;
  const wrongAnswerCount = totalQuestions - correctAnswerCount
```